.gitignore

```diff
@@ -0,0 +1,11 @@
+# binary files
+protoc
```

Comment:
these are installed inside the folder name called google right?

Reply:
no, they are directly inside our repo

Reply:
ew, better not do that